### ISSUE TITLE
ftdidb_lock while trying to remotely connect to hw_server

### ISSUE THREAD

**Issue by bentomo** [stv0g/xilinx-hw-server-docker#5]
I followed the instructions and it appears I can connect to the pi from a remote XSDB section. The session connects without reporting any error. However, when I try to list targets to a known working Xilinx eval board I get the following error:

`ftdidb_lock failed: FTDMGR wasnt properly initialized`

I get the same message with a digilent HS3 usb jtag adapter and a Xilinx Platform 2 USB cable. I will experiment and see if I can get it working. I'm using a Raspberry pi model 3B+ on buster 64 bit, kernel 5.15.32-v8+

To reproduce - 
```
xsct
connect -url <piaddress>:3121
targets
ftdidb_lock failed: FTDMGR wasnt properly initialized
```

Quick aside this may be missing from the instructions as well otherwise the docker containers won't start in usermode
`sudo chmod 666 /var/run/docker.sock`

**Comment by bentomo** [comment]
For the root access, I had other issues following the instructions from docker, but that's probably due to my limited docker experience, we can table that one for now.

 However, I did have better success using the 2020.1 version after a few tries. I think it came down to a version mismatch. I was trying to connect from a 2020.2 xsct session to the 2021 docker container. After downgrading to the prepackaged 2020.1, I was able to download a bitstream and even get a litex UART output!!!! This is amazing! Good bye smart lynq!
![image](https://user-images.githubusercontent.com/5197817/173848193-ebea6ecf-8c89-4808-84ee-83c2933a9069.png)

I did try a few other things in between however, one thing was including the udev rule from digilent for the hs3 and I don't know if that made a difference or not. I will have to package up my own following more instructions and get this auto starting on the pi. WooHOO!
